src/visualization/components/Player.tsx

```diff
@@ -156,14 +165,16 @@ export default connect(
             playback: state.playback,
             speed: state.speed,
             view: state.view,
-            fullpage: state.fullpage            
+            fullpage: state.fullpage,
+           
```

Comment:
unnecessary, same below

Reply:
not resolved - if you disagree mark as won'tfix and give an explanation

---
In reply to: [175851128](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175851128) [](ancestors = 175851128)